### PR TITLE
populate community_translations with transaction_type_translations 

### PR DESCRIPTION
**Pull request by Gnito**
+ indexes for community_id and translation_key

### CODE REVIEW

File: app/models/community_translation.rb [hunk 1]
```diff
@@ -10,6 +10,11 @@
 #  created_at      :datetime         not null
 #  updated_at      :datetime         not null
 #
+# Indexes
+#
+#  index_community_translations_on_community_id     (community_id)
+#  index_community_translations_on_translation_key  (translation_key)
```
Comment (Gnito) [inline app/models/community_translation.rb]: Hmm. locale is even more important. 
Maybe the best way is making index for multiple keys (community_id + locale)

File: db/migrate/20150312155301_insert_transaction_type_translations_to_community_translations.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
+class InsertTransactionTypeTranslationsToCommunityTranslations < ActiveRecord::Migration
+  def up
+    # Index by community_id and translation_key
+    add_index :community_translations, :community_id            unless index_exists?(:community_translations, :community_id)
+    add_index :community_translations, [:community_id, :locale] unless index_exists?(:community_translations, [:community_id, :locale])
+    add_index :community_translations, :translation_key         unless index_exists?(:community_translations, :translation_key)
```
Comment (Gnito) [inline db/migrate/20150312155301_insert_transaction_type_translations_to_community_translations.rb]: I guess, this should be combined index too?

**Comment by rap1ds** [comment]
:+1: Test the migration on staging before deploying to prod

**Comment by Gnito** [comment]
I added write functionality for translation service. (in TransactionTypeCreator)

**Comment by rap1ds** [comment]
:+1:

**Pull request merged**